code/game/gamemodes/blob/overmind.dm

```diff
@@ -31,6 +31,7 @@
 	if(blob_core)
 		blob_core.adjustcolors(blob_reagent_datum.color)
 
+	color = blob_reagent_datum.complementary_color
```

Comment:
I don't get the purpose of this.

Reply:
It sets a color of the Blob marker when the Blob first spawns and it's reagent is randomly rolled. I added complementary colors, because if marker used the same color as the rest of the Blob, it would be hard to see for the player.